guilford_court_commitment.yaml

```diff
@@ -346,9 +303,13 @@ associationDefinitions:
       ol.id:
         type: "ol.id"
         transforms:
-        - !<transforms.HashTransform>
-          columns: ["COMMITTED FIRST NAME", "COMMITTED MIDDLE NAME", "COMMITTED LAST NAME", "COMMITTED NAME SUFFIX", "OFFENDER NC DOC ID NUMBER", "MOST SERIOUS OFFENSE CODE"]
-          hashFunction: "sha256"
+        - !<transforms.ConcatCombineTransform>
+          transforms:
+          - !<transforms.HashTransform>
+            columns: ["OFFENDER NC DOC ID NUMBER", "MOST SERIOUS OFFENSE CODE"]
```

Comment:
I don't know if it matters, but here the two columns were listed in reverse and then hashed - on the safe side maybe match that here. https://github.com/Lattice-Works/Guilford-Co-NC/blob/master/guilford_inmates_dps.yaml#L747